apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -126,6 +126,23 @@ internal bool HasMutations
             }
         }
 
+        private Priority _commitPriority;
+
+        /// <summary>
+        /// The RPC priority to use for the commit RPC of this transaction. This can only be set for read/write transactions.
```

Comment:
Consider adding here what it deafaults to? Or making it nullable, I think this is not a required field, right? Make sure to add a test to check for its initial value (whtether a default or `null`) so that we don't inadvertenly change it over time. Here and in commands as well.

Reply:
I've added comments and tests for the default. The value will default to `Unspecified`, which is also one of the possible values in the proto enum. I don't think we should make this nullable, as it would mean that we would either have two ways of setting unspecified, or that we would not have a one-to-one mapping between the Spanner.Data.Priority enum and the proto enum. The property is not nullable in the `RequestOptions` proto.

Reply:
OK